doc/colorscheme-primary.txt

```diff
@@ -42,6 +42,9 @@ does not support 256 colors. No problem - just remove that line and consult
 If you want to disable italic font, add this line to your .vimrc: >
   let g:colorscheme_primary_disable_italic = 1
 <
+                                        *g:colorscheme_primary_bg_transparency*
+If you want to enable background transparency, add this line to your .vimrc: >
+  let g:colorscheme_primary_bg_transparency = 1
```

Comment:
Syntax nit: Isn't this missing the `<` to explicitly mark the end of the block?

Reply:
Yep, I'll fixed later, thanks!

Reply:
> Syntax nit: Isn't this missing the `<` to explicitly mark the end of the block?

Hi, I already fixed it, and have changed the option to `g:colorscheme_primary_enable_transparent_bg`, please review the codes again, thanks!